hybrid/clusterlink/mtls_source_cluster/README.md

```diff
@@ -100,7 +100,7 @@ kubectl -n destination create secret generic rest-credential \
     --from-file=basic.txt=$TUTORIAL_HOME/rest-credential.txt
     
 kubectl -n destination create secret generic password-encoder-secret \
-    --from-file=password_encoder_secret=$TUTORIAL_HOME/password-encoder-secret.txt
+    --from-file=password-encoder.txt=$TUTORIAL_HOME/password-encoder-secret.txt
```

Comment:
@ptyagii Did this work?
Format for password encode file is different. Check https://github.com/confluentinc/confluent-kubernetes-examples/blob/master/hybrid/schemalink/mtls/password-encoder-secret.txt

Can you update the resources for this examples?

Reply:
I didn't run the test, but tried to keep it the same as examples for ccloud [ccloud-as-destination-cluster](https://github.com/confluentinc/confluent-kubernetes-examples/tree/master/hybrid/clusterlink/ccloud-as-destination-cluster) and [ccloud-as-source-cluster](https://github.com/confluentinc/confluent-kubernetes-examples/tree/master/hybrid/clusterlink/ccloud-as-source-cluster). 

+1 to the password format, updated the format in `password-encoder-secret.txt` file